lib/formulator.ex

```diff
@@ -26,13 +26,23 @@ defmodule Formulator do
       #=> <input id="user_name" name="user[email]" aria-label="email" type="text" value="">
 
   Passing other options:
-      <%= input form, :email, label: [class: "control-label"] %>
+      <%= input form, :email, as: :email, label: [class: "control-label"] %>
```

Comment:
This is to show the `label` part, not the `as:`. For now I'd say let's either leave it alone or if it really bothers you, change the email to be "name".